src/components/application_manager/rpc_plugins/app_service_rpc_plugin/src/app_service_rpc_plugin.cc

```diff
@@ -107,14 +107,14 @@ void AppServiceRpcPlugin::DeleteSubscriptions(
 
 }  // namespace app_service_rpc_plugin
 
-extern "C" __attribute__((visibility("default")))
-application_manager::plugin_manager::RPCPlugin*
-Create() {
+using RPCPlugin = application_manager::plugin_manager::RPCPlugin;
```

Comment:
Why keep these Create and Delete functions?

Reply:
This functions are needed to handle dynamic shared libraries (in this particular case AppServiceRpcPlugin), and in fact they are used and were marked as unused by Cppcheck by mistake.

Please look at example of usage of this functions: https://github.com/smartdevicelink/sdl_core/blob/master/src/components/application_manager/src/plugin_manager/rpc_plugin_manager_impl.cc#L62